src/modules/chat/chat-view.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import WithDrawerLayout from 'core/components/layout/with-drawer-layout';
+import { Channel } from './types';
+import ChatService from './chat-service';
+import ChatChannels from './components/chat-channels';
+import ChatConversation from './components/chat-conversation';
+
+const ChatView: React.FC = () => {
+  const [channels, setChannels] = React.useState<Channel[]>([]);
+  const [currentChannel, setCurrentChannel] = React.useState<Channel>();
+
+  const fetchChannels = React.useCallback(async () => {
+    try {
+      const data = await ChatService.fetchChannels();
+      setChannels(data);
+      setCurrentChannel(data[0]);
```

Comment:
Change setting current channel depending on last selected channel from localStorage, and set current channel to first possible if there's no selected channel in localStorage